release/scripts/perform.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+
+#set -x
+
+BASE_BRANCH="main"
+
+NOW_UTC=$(date -u '+%Y%m%d%H%M%S')
+RELEASE_BRANCH="release-$NOW_UTC"
+
+ensure_release_branch() {
+  local STATUS=$(git status -s | grep -v '??')
+
+  if [ "$STATUS" != "" ]; then
+    >&2 echo "ERROR Dirty working copy found! Stop."
+    exit 1
+  fi
+
+  git switch -c ${RELEASE_BRANCH} ${BASE_BRANCH}
+  git push -u origin ${RELEASE_BRANCH}
```

Comment:
Can we leave out the origin here? Not toally sure, but I think that should work. If the current branch that the user was on tracked a branch on the stackable github repo, the new branch should be set up to track that remote as well and be pushed there automatically.
My remotes are not always called origin, so at least for me this may fail - but I can change that if I cause issues with this behavior :)

Reply:
I don't think it's possible to leave the repository argument out when pushing just one branch. specially when you have multiple remotes configured. I've extracted it into a variable for now.